### ISSUE TITLE
Bootstrapping a new project is broken

### ISSUE THREAD

**Issue by icemac** [flyingcircusio/batou#163]
https://pypi.org/project/batou/ says:

Getting started with a new project is easy:

```
mkdir myproject
cd myproject
git init
curl -sL https://raw.githubusercontent.com/flyingcircusio/batou/master/bootstrap | sh
```

This break with:

```
$ curl -sL https://raw.githubusercontent.com/flyingcircusio/batou/master/bootstrap | sh
Running unclean installation from requirements.txt
Creating venv ...
Ensuring pip ...
Ensuring unclean install ...
usage: batou [-h] [-d] {deploy,secrets} ...
batou: error: invalid choice: 'appenv-update-lockfile' (choose from 'deploy', 'secrets')
```

This is probably related to #162.

**Comment by icemac** [comment]
#151 might also fix this problem. (Not checked this personally.)

**Comment by ctheune** [comment]
Fixed in #167
